lib/src/Picker/Picker.tsx

```diff
@@ -35,6 +35,8 @@ export type ToolbarComponentProps<
   timeIcon?: React.ReactNode;
   isLandscape: boolean;
   ampmInClock?: boolean;
+  mobileKeyboardViewIcon?: React.ReactNode;
+  mobileDefaultViewIcon?: React.ReactNode;
```

Comment:
Probably better name will be 
```suggestion
  mobilePickerViewIcon?: React.ReactNode;
```

Reply:
And also if it is a public API it should be documented in jsdoc. I am not sure where we are documenting props for `Toolbar` but please try just adding here a comment with description